### DISCUSSION TITLE
Automatic Internal Post Linking

### DISCUSSION THREAD

**Discussion by scotdill** [GetPublii/Publii#1418] [Ideas]
Hey Publii,
Internal links are big for SEO.
It would be amazing if every article we could outline a few keywords that we want the autolinking to work for, then, for every new post that publishes if the words match up on that specific article then they are automatically linked.
That is my idea of implementation, but if you have any other ideas that are even better, would love to have that.
I do think this is important.
Do you think this is a possiblity?

**Comment by example-user30** [comment]
This might be a good use to create a plugin to do the heavy lifting.  You could have the replacement function used in imageDecoding Plugin do the heavy lifting.  You would need to design a front end for the admin side to manage the key words.  That might take a bit more work to figure out.
